std/experimental/allocator/package.d

```diff
@@ -226,6 +226,22 @@ module std.experimental.allocator;
 public import std.experimental.allocator.common,
     std.experimental.allocator.typed;
 
+shared static this()
+{
+    import std.experimental.allocator.gc_allocator : GCAllocator;
+    _processAllocator = sharedAllocatorObject(GCAllocator.instance);
+}
+
+unittest
```

Comment:
std/experimental/allocator/package.d(235:1)[warn]: A unittest should be annotated with at least @safe or @system